.env.default

```diff
@@ -10,6 +10,6 @@
 HTTP_SERVER_PORT=7272
 HTTP_SERVER_HOST=0.0.0.0
 
-CONTENT_SERVER_ADDRESS=
+CONTENT_SERVER_ADDRESS=https://peer-ec1.decentraland.org/content
```

Comment:
This should be undone. Please use `.env` that is not committed to the repo to avoid pushing it.